bokun.yaml

```diff
@@ -366,8 +366,8 @@ paths:
       tags:
         - currency
       operationId: allCurrencies
-      summary: Get all Supported Currencies
-      description: 'Get a list of all the available currencies, along with conversion rates.'
+      summary: Get all system Supported Currencies
```

Comment:
I think `system` is redundant here, since `supported` already implies the system. Same for two similar changes down below.